docs/_includes/component.njk

```diff
@@ -1,434 +1,376 @@
-        </tr>
-      </thead>
-      <tbody>
-        {% for cssPart in component.cssParts %}
-          <tr>
-            <td class="nowrap">
-              <code>{{ cssPart.name }}</code>
-            </td>
-            <td>{{ cssPart.description | markdownInline | safe }}</td>
-          </tr>
-        {% endfor %}
-      </tbody>
-    </table>
+<table>
+  <thead>
+    <tr>
+      <th class="table-name">Name</th>
+      <th class="table-description">Description</th>
+    </tr>
+  </thead>
+  <tbody>
+    {% for cssPart in component.cssParts %}
+    <tr>
+      <td class="nowrap">
+        <code>{{ cssPart.name }}</code>
+      </td>
+      <td>{{ cssPart.description | markdownInline | safe }}</td>
+    </tr>
+    {% endfor %}
+  </tbody>
+</table>
 
-    <p>
-      <em
+<p>
+  <em
     >Learn more about <a href="{{ rootUrl('/getting-started/customizing/#css-parts') }}">customizing CSS parts</a>.</em
   >
-    </p>
-  {% endif %}
-  {# Animations #}
-  {% if component.animations.length %}
-    <h2>Animations</h2>
+</p>
+{% endif %} {# Animations #} {% if component.animations.length %}
+<h2>Animations</h2>
 
-    <table>
-      <thead>
-        <tr>
-          <th class="table-name">Name</th>
-          <th class="table-description">Description</th>
-        </tr>
-      </thead>
-      <tbody>
-        {% for animation in component.animations %}
-          <tr>
-            <td class="nowrap">
-              <code>{{ animation.name }}</code>
-            </td>
-            <td>{{ animation.description | markdownInline | safe }}</td>
-          </tr>
-        {% endfor %}
-      </tbody>
-    </table>
+<table>
+  <thead>
+    <tr>
+      <th class="table-name">Name</th>
+      <th class="table-description">Description</th>
+    </tr>
+  </thead>
+  <tbody>
+    {% for animation in component.animations %}
+    <tr>
+      <td class="nowrap">
+        <code>{{ animation.name }}</code>
+      </td>
+      <td>{{ animation.description | markdownInline | safe }}</td>
+    </tr>
+    {% endfor %}
+  </tbody>
+</table>
 
-    <p>
-      <em
+<p>
+  <em
     >Learn more about <a href="{{ rootUrl('/getting-started/customizing#animations') }}">customizing animations</a>.</em
   >
-    </p>
-  {% endif %}
-  {# Dependencies #}
-  {% if component.dependencies.length %}
-    <h2>Dependencies</h2>
+</p>
+{% endif %} {# Dependencies #} {% if component.dependencies.length %}
+<h2>Dependencies</h2>
 
-    <p>This component automatically imports the following dependencies.</p>
+<p>This component automatically imports the following dependencies.</p>
 
-    <ul>
-      {% for dependency in component.dependencies %}
-        <li>
-          <code>&lt;{{ dependency }}&gt;</code>
-        </li>
-      {% endfor %}
-    </ul>
-  {% endif %}
-{% endblock %}
\ No newline at end of file
+<ul>
+  {% for dependency in component.dependencies %}
+  <li>
+    <code>&lt;{{ dependency }}&gt;</code>
+  </li>
+  {% endfor %}
+</ul>
+{% endif %} {% endblock %}
```

Comment:
No, the spacing & indentation was just VS code being weird (see my other comment about the nunjucks formatting). The change here was to the "Guidelines" section (line 81) -- I realized the anchor links were not working properly (because of the way the nunjucks file is setup, I think?) for the the Guidelines subheadings, so I just switched them to have one "Usage Guidelines" header with just bolded section titles below.